.gitignore

```diff
@@ -1,5 +1,8 @@
 /target
 /.idea
+/.vscode/**
+!/.vscode/launch.json
+!/.vscode/settings.json
```

Comment:
I think we should keep these files. The spelling errors and launch configurations can be shared this way.

Reply:
I want to ignore other files in .vscode dir. Those files are still in git